vignettes/worsening_vignette/readme.md

```diff
@@ -34,7 +34,7 @@ Gaile, a resident anesthesiologist at Midwest Medicine, has the following perfor
 |Mar  |              93%|               96|          91|
 
 ## Preference data
-Preferences for precision feedback are elicited through a preference survey that providers can take. The preference survey produces a preference model for each provider that, with the provider's permission, is shared with MPOG to maintain. MPOG analyses preferences data that is shared to identify population-level preference segments. These are generates as preference profiles that can serve as a default preference model for an organization, or which can be selected by providers who do not take the preference survey, but who identify preferences that are close enough to their own in the settings menu for the precision feedback system.
+Preferences for precision feedback are elicited through a preference survey that providers can take. The preference survey produces a preference model for each provider that, with the provider's permission, is shared with MPOG to maintain. MPOG analyses preferences data that is shared to identify population-level preference segments. This generates preference profiles that can serve as a default preference model for an organization, or which can be selected by providers who do not take the preference survey, but who identify preferences that are close enough to their own in the settings menu for the precision feedback system.
```

Comment:
Changed the wording here on only this vignette before I decided to elevate this to an issue for team review, per feedback from ZLL on 5/30

See issue #104

Reply:
Wording updated to social loss with Zach 6/6, propagating out in new issue #104